src/iris/api.py

```diff
@@ -1665,7 +1764,41 @@ def on_get(self, req, resp):
         query parameter can be used. The fields parameter takes the value of a comma-separated list of attributes
         (e.g. id,owner), and the API will only include these incident fields in the output. If no "fields" value is
         specified, all fields will be returned.
+
+        for querying tags a "tag_" prefix should be added before the tag name and then it can be used just as any other normal field
+        e.g. if we want to query for an incident that has a tag "service" with a value "foo" we would query: /v0/incidents?tag_service=foo
+
+        Additionally adding the parameter "counts" will return a dictionary with unique values of each field and their counts as well as a total count
+
+        **Example request**:
+
+        .. sourcecode:: http
+
+            GET /v0/incidents?created__ge=1700693640&fields=owner&fields=application&counts=true  HTTP/1.1
+            Host: example.com
+
+        **Example response**:
+
+        .. sourcecode:: http
+
+            HTTP/1.1 200 OK
+            Content-Type: application/json
+
+            {
+                "field_counts": {
+                    "owner": {
+                        "foo_user": 4,
+                        "None": 7
+                    },
+                    "application": {
+                        "Autoalerts": 11
+                    }
+                },
+                "total_count": 11
+            }
         '''
+        counts_only = req.get_param('counts')
```

Comment:
Would it be possible for the other get_param calls added in this PR also get converted to get_param_as_bool?

Reply:
ah, yeah